.github/scripts/test_check_labels.py

```diff
@@ -95,11 +95,9 @@ def test_ci_fails_without_required_labels(
         mock_parse_args: Any,
```

Comment:
The test name should be updated since we no longer expect the CI to fail

Something like `test_when_pr_needs_lable_then_leave_comment"

Reply:
Ah good catch!